content/pages/FAQ.rst

```diff
@@ -15,7 +14,13 @@ How are topics chosen?
 Most session topics are scheduled the morning of the event and are provided by
 the attendees.
 
---------------
+What are Lightning Talks?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Lightning talks are five minute presentations on any topic. At barcamp,
+the first session will be just lightning talks in the barcamp style;
```

Comment:
My assumption the last time we talked about this was that the lightening talks are happening during the first session along with the other talks. Having it by itself is not going to go well with people.